sites/main-site/src/content/events/2025/hackathon-march-2025/san-francisco-bay-area.mdx

```diff
@@ -1,19 +1,23 @@
 ---
 title: "Hackathon - March 2025 (San Francisco Bay Area)"
-subtitle: "Local node of the nf-core hackathon in San Francisco (Location TBD)"
+subtitle: "Local node of the nf-core hackathon in Downtown San Francisco"
 shortTitle: "🇺🇸 San Francisco Bay Area"
 type: "hackathon"
 startDate: "2025-03-24"
 startTime: "09:00-08:00"
 endDate: "2025-03-26"
 endTime: "17:00-08:00"
 locations:
-    - name: To Be Determined
+    - name: Amazon | Downtown San Francisco
       address: |
-          To Be Determined
+          SFO28
+          525 Market St
+          San Francisco, CA 94105 United States
       links:
           - https://nf-co.re/events/2025/hackathon-march-2025
```

Comment:
```suggestion
```
No need for this. These links are meant to be just the location really - they show up in the header next to the location name.

All hackathon event sub-pages have a box linking back to the main page already.